Source/CarthageKit/Git.swift

```diff
@@ -136,6 +136,34 @@ extension Submodule: CustomStringConvertible {
 	}
 }
 
+/// Struct to encapsulate global fetch interval cache
+public struct FetchCache {
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
```

Comment:
This doesn't have value semantics since the entire interface is static. Very misleading: every user of this type is actually implicitly coupled now 👎